src/retroshare/rsposted.h

```diff
@@ -213,16 +288,55 @@ class RsPosted : public RsGxsIfaceHelper, public RsGxsCommentService
 //virtual bool createNewComment(uint32_t &token, RsGxsComment &comment) = 0;
 //virtual bool createNewVote(uint32_t &token, RsGxsVote &vote) = 0;
 
-    RS_DEPRECATED_FOR(setPostReadStatus)
-    virtual void setMessageReadStatus(uint32_t& token, const RsGxsGrpMsgIdPair& msgId, bool read) = 0;
+	/**
+	 * @brief toggle message read status
+	 * @deprecated
+	 * @param[out] token GXS token queue token
+	 * @param[in] msgId
+	 * @param[in] read
+	 */
+	RS_DEPRECATED_FOR(setPostReadStatus)
+	virtual void setMessageReadStatus(uint32_t& token, const RsGxsGrpMsgIdPair& msgId, bool read) = 0;

```

Comment:
There shouldn't any JSON API for themethods with token, because they are not compatible.

Reply:
i can disable it

Reply:
but that is always didabled it i did only what @G10h4ck  did

Reply:
i added for readstatus a api but @sehraf said it not needed it yet,

Reply:
i has removed for readstatus the api code